jccli/cli/system.py

```diff
@@ -0,0 +1,81 @@
+import json
+import click
+from jccli.jc_api_v1 import JumpcloudApiV1
+
+
+@click.group()
+@click.pass_context
+def system(ctx):
+    """
+    Command set for systems
+    """
+    pass
+
+
+@system.command("get")
+@click.option('--id', '-i', 'system_id', required=True, type=str)
+@click.pass_context
+def get_system(ctx, system_id):
+    """
+    Detail view of system, outputted in JSON.
```

Comment:
i would remove output info from docs since output type can be independent of the command.

Reply:
done